Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -369,76 +386,76 @@ class GymDetailViewController: UIViewController, UITableViewDelegate, UITableVie
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if(tableView == hoursTableView){
-            let cell = tableView.dequeueReusableCell(withIdentifier: "gymHoursCell", for: indexPath) as! GymHoursCell
+            let cell = tableView.dequeueReusableCell(withIdentifier: GymHoursCell.identifier, for: indexPath) as! GymHoursCell
             let date = Date()
             let day = (date.getIntegerDayOfWeekToday()! + indexPath.row + 1)%7
             
-            switch day{
-            case 0:
+            switch days[day]{
+            case .sunday:
                 cell.dayLabel.text = "Su"
                 if let gymHours = gym.gymHours.zero{
                     cell.hoursLabel.text = gymHours.openTime + " - " + gymHours.closeTime
```

Comment:
I believe it is cleaner if you do `cell.hoursLabel.text = "\(gymHours.openTime) - \(gymHours.closeTime)"`

Reply:
Also this is a lot of duplicated code. Maybe it makes sense to have a helper function like `func getHoursForGym(gym: Gym) -> String` and do the logic once in there